go-controller/pkg/types/const.go

```diff
@@ -197,4 +197,7 @@ const (
 	// entry for the gateway routers. After this time, the entry is removed and
 	// may be refreshed with a new ARP request.
 	GRMACBindingAgeThreshold = "300"
+
+	APBRouteErrorMsg         = "failed to apply policy"
+	EgressFirewallApplyError = "EgressFirewall Rules not correctly applied"
```

Comment:
I am not sure if defining those as `error` is useful, because those are never used in the code as `error`. Usually there is another `error` returned by the handler, and these messages are just inserted to the status message for more context. I could make some `error_msg` file and put all error messages there maybe?

Reply:
maybe status.go?

Reply:
or resource_status.go